INSTALL.md

```diff
@@ -3,82 +3,94 @@
+
 You must first install development and libraries to your host.
 Assuming Ubuntu:
 
-```
+```bash
 $ sudo apt-get update && \
-sudo apt-get install -y build-essential \
-libssl-dev uuid-dev libgpgme11-dev squashfs-tools libseccomp-dev pkg-config
+  sudo apt-get install -y build-essential \
+  libssl-dev uuid-dev libgpgme11-dev squashfs-tools libseccomp-dev pkg-config
 ```
 
 On CentOS/RHEL:
 
+```bash
+$ sudo yum groupinstall -y 'Development Tools' && \
+  sudo yum install -y openssl-devel libuuid-devel libseccomp-devel
 ```
-$ sudo yum groupinstall -y 'Development Tools'
-$ sudo yum install -y openssl-devel libuuid-devel libseccomp-devel
-```
-Skip libseccomp-devel on CentOS/RHEL 6.
+Skip `libseccomp-devel` on CentOS/RHEL 6.
 
 ## Install golang
 
 This is one of several ways to [install and configure golang](https://golang.org/doc/install).
 
 First, visit the [golang download page](https://golang.org/dl/) and pick a
-package archive to download.  Copy the link address and download with `wget`.
-
-```
-$ export VERSION=1.11 OS=linux ARCH=amd64
-$ cd /tmp
-$ wget https://dl.google.com/go/go$VERSION.$OS-$ARCH.tar.gz
-```
+package archive to download. Copy the link address and download with `wget`.
 
 Then extract the archive to `/usr/local` (or use other instructions on go
 installation page).
 
-```
-$ sudo tar -C /usr/local -xzf go$VERSION.$OS-$ARCH.tar.gz
-```
+```bash
+$ export VERSION=1.11.4 OS=linux ARCH=amd64
 
-Finally, set up your environment for go
+$ cd /tmp  # or wherever you want to store golang
 
+$ wget https://dl.google.com/go/go$VERSION.$OS-$ARCH.tar.gz && \
+  sudo tar -C /usr/local -xzf go$VERSION.$OS-$ARCH.tar.gz
 ```
-$ echo 'export GOPATH=${HOME}/go' >> ~/.bashrc
-$ echo 'export PATH=/usr/local/go/bin:${PATH}:${GOPATH}/bin' >> ~/.bashrc
-$ source ~/.bashrc
+
+Finally, set up your environment for Go:
+
+```bash
+$ echo 'export GOPATH=${HOME}/go' >> ~/.bashrc && \
+  echo 'export PATH=/usr/local/go/bin:${PATH}:${GOPATH}/bin' >> ~/.bashrc && \
+  source ~/.bashrc
 ```
 
 ## Clone the repo
 golang is a bit finicky about where things are placed. Here is the correct way
-to build Singularity from source.
+to build Singularity from source:
 
-```
-$ mkdir -p $GOPATH/src/github.com/sylabs
-$ cd $GOPATH/src/github.com/sylabs
-$ git clone https://github.com/sylabs/singularity.git
-$ cd singularity
+```bash
+$ mkdir -p ${GOPATH}/src/github.com/sylabs && \
+  cd ${GOPATH}/src/github.com/sylabs && \
+  git clone https://github.com/sylabs/singularity.git && \
+  cd singularity
 ```
 
 ## Compile the Singularity binary
 Now you are ready to build Singularity. Dependencies will be automatically
 downloaded. You can build Singularity using the following commands:
 
-```
-$ cd $GOPATH/src/github.com/sylabs/singularity
-$ ./mconfig
-$ cd ./builddir
-$ make
-$ sudo make install
+```bash
+$ cd ${GOPATH}/src/github.com/sylabs/singularity && \
+  ./mconfig && \
+  cd ./builddir && \
+  make && \
+  sudo make install && \
```

Comment:
There is a syntax error here.  The final line should not end with an `&& \`.  Please actually run all of these commands in a virtual machine to make sure they actually work before suggesting users do it.

Reply:
Good catch! I'll make sure that won't happen again!